src/gc/gcimpl.h

```diff
@@ -6,7 +6,7 @@
 #ifndef GCIMPL_H_
 #define GCIMPL_H_
 
-#define CLREvent CLREventStatic
+#define CLREvent GCEvent
```

Comment:
Comments that say "do not ever change this" without explaining the reason are meant to be ignored.